recipe/meta.yaml

```
@@ -50,6 +57,7 @@ outputs:
         - pkg-config  # [not win]
         - unzip  # [not win]
         - make  # [not win]
+        - sed  # [osx]
```

Since this is only needed on the **osx-arm64** worker, we can be more explicit here.
```suggestion
        - sed  # [osx and arm64]
```

---

Fixed.